File: OLSource1/v1/VS_officedev/codesnippet/CSharp/specific-security-considerations-for-office-solutions_1.cs
        private void UntrustedCode()
        {
            Microsoft.Office.Interop.Outlook.Application application =
                new Microsoft.Office.Interop.Outlook.Application();
            Microsoft.Office.Interop.Outlook.MailItem mailItem1 =
                application.CreateItem(
                Microsoft.Office.Interop.Outlook.OlItemType.olMailItem) as
                Microsoft.Office.Interop.Outlook.MailItem;
            mailItem1.To = "someone@example.com";
            MessageBox.Show(mailItem1.To);
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/specific-security-considerations-for-office-solutions_1.vb
    Private Sub UntrustedCode()
        Dim application As New Microsoft.Office.Interop.Outlook.Application
        Dim mailItem1 As Microsoft.Office.Interop.Outlook.MailItem = _
            TryCast(application.CreateItem( _
            Microsoft.Office.Interop.Outlook.OlItemType.olMailItem),  _
            Microsoft.Office.Interop.Outlook.MailItem)
        mailItem1.To = "someone@example.com"
        MessageBox.Show(mailItem1.To)
    End Sub